book/LocalContextsPlaceholder/projects.md

```diff
@@ -0,0 +1,104 @@
+# Local Contexts Projects
+
+## About
+On the [Local Contexts Hub](https://localcontextshub.org), we use the term "Local Contexts Projects" to describe the context where Labels or Notices are being applied. Examples of Local Contexts Projects include: a website, university syllabus, dissertation, publication, dataset, museum exhibition, archival record, item in a collection, library database, photography collection, voucher specimen, or metadata about a record.
+
+## Example: Local Contexts Project Graph
+The following graph represents a basic record we might use for a Local Contexts Project.
+
+As Ocean InfoHub is leveraging Schema.org we are using [schema.org/Project](https://schema.org/Project) for this type. Any of the properties of Project seen there are valid to use in such a record.
+
+```{literalinclude} ../../../odis-in/dataGraphs/thematics/projects/graphs/local-contexts-project-example.json
```

Comment:
```suggestion
```{literalinclude} ../../../odis-in/dataGraphs/thematics/projects/graphs/localContexts-project-example.json
```
to match the current changes in the PR in the `odis-in` repo